test/include/utils.hpp

```diff
@@ -131,3 +143,15 @@ static rgl_mesh_t loadMesh(std::filesystem::path path)
 	EXPECT_RGL_SUCCESS(rgl_mesh_create(&mesh, vs.data(), vs.size(), is.data(), is.size()));
 	return mesh;
 }
+
+static std::vector<TestPointStruct> GenerateTestPointsArray(int count, rgl_mat3x4f transform = identityTestTransform)
+{
+        std::vector<TestPointStruct> points;
+        for (int i = 0; i < count; ++i) {
+            auto currentPoint = TestPointStruct { .xyz = { i, i + 1, i + 2 }, .isHit = i % 2, .intensity = 100 };
+            currentPoint.xyz = Mat3x4f::fromRGL(transform) * currentPoint.xyz;
+            points.emplace_back(currentPoint);
+
+        }
+        return points;
+}
```

Comment:
It could be moved to `RGLGraphTest`/`RGLPointFieldsTest` struct when you will apply the previous comment.